pkg/adapter/client_unix.go

```diff
@@ -14,7 +14,14 @@ func formatDefaultBridge(remoteConn *remoteclientconfig.RemoteConnection, logLev
 	if port == 0 {
 		port = 22
 	}
+	options := ""
+	if remoteConn.IdentityFile != "" {
+		options += " -i " + remoteConn.IdentityFile
+	}
+	if remoteConn.IgnoreHosts {
+		options += " -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+	}
 	return fmt.Sprintf(
-		`ssh -p %d -T %s@%s -- /usr/bin/varlink -A \'/usr/bin/podman --log-level=%s varlink \\\$VARLINK_ADDRESS\' bridge`,
-		port, remoteConn.Username, remoteConn.Destination, logLevel)
+		`ssh -p %d -T%s %s@%s -- varlink -A \'podman --log-level=%s varlink \\\$VARLINK_ADDRESS\' bridge`,
```

Comment:
@afbjorklund Is there a reason for pulling the fixed paths?

Reply:
Pulling? I removed the hard-coded paths, that missed /usr/local/bin